Source/ACE/Command/Handlers/AdminCommands.cs

```diff
@@ -1001,8 +1001,26 @@ public static void HandleCI(Session session, params string[] parameters)
                 ChatPacket.SendServerMessage(session, "Not a valid weenie id - must be a number between 0 -65,535 ", ChatMessageType.Broadcast);
                 return;
             }
-            var loot = LootGenerationFactory.CreateTestWorldObject(session.Player, weenieId);
-            session.Player.HandleAddToInventory(loot);
+
+            // TODO: we have to be able to identify containers - you can create a wo and see if it has
+            // an item capacity but that seems a bit wasteful.   TBD on best method.   Limited number so
+
+            if (weenieId == 136)
+            {
+                var loot = LootGenerationFactory.CreateTestContainerObject(session.Player, weenieId);
+                session.Player.AddToInventory(loot);
```

Comment:
You shouldn't modify a player directly from the command/network based handler -- it races with the player accessing/modifying itself.  This is what actions are for.  Instead, enqueue an action on the player to deal with their own data.